Zend/tests/property_hooks/final_prop.phpt

```diff
@@ -1,13 +1,12 @@
 --TEST--
-errmsg: properties cannot be final
+Final property is not allowed
 --FILE--
 <?php
 
-class test {
-    final $var = 1;
+class A {
+    public final $prop { get {} set {} }
```

Comment:
The RFC says:

> A property may also be declared final. 

But the test seems to be doing the opposite?

Reply:
Thanks. We removed final for a short while, I added this to my TODO list.